lib/event_associator.rb

```diff
@@ -0,0 +1,7 @@
+module EventAssociator
```

Comment:
I don't know if I would put this in `lib`. This has knowledge of the models, and therefore of the business logic, so I would put it in `app` (as a part of a service or something).

Reply:
I guess i can put it in the concern folder, what do you think?